src/nurse/utilities.py

```diff
@@ -0,0 +1,9 @@
+def get_users_for_prescriptions(prescriptions):
```

Comment:
why not making it part of the _notifications module like you did originally since we only use it there at the moment?